sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVOptions.scala

```diff
@@ -161,12 +163,3 @@ private[csv] class CSVOptions(@transient private val parameters: CaseInsensitive
     settings
   }
 }
-
-object CSVOptions {
```

Comment:
The `CSVOptions` (and also `JSONOptions`) will always have to take `timeZone` option.
I don't want callers to forget to specify it by these convenient methods.
Or should I add the default timezone id to these methods?